doc/contributing/testing_ruby.md

```diff
@@ -20,6 +20,12 @@ We can run any of the make scripts [in parallel](building_ruby.md#label-Running+
     make btest OPTS=-v
     ```
 
+    To run an individual bootstrap test, we can set the filename in the environment variable `BTESTS`:
+
+    ```
+    make btest BTESTS=bootstraptest/test_gc.rb
```

Comment:
`make btest BTESTS=--sets=method` also work.

Reply:
It's not "the environment variable".

Reply:
@peterzhu2118 ?

Reply:
I opened #6194.